docs/src/main/sphinx/connector/kudu.rst

```diff
@@ -228,67 +229,55 @@ The data types of Trino and Kudu are mapped as far as possible:
-| ``ALTER TABLE ... RENAME COLUMN ...``    | Only allowed, if not part of  |
-|                                          | primary key                   |
-+------------------------------------------+-------------------------------+
-| ``ALTER TABLE ... ADD COLUMN ...``       | See `Add Column`_             |
-+------------------------------------------+-------------------------------+
-| ``ALTER TABLE ... DROP COLUMN ...``      | Only allowed, if not part of  |
-|                                          | primary key                   |
-+------------------------------------------+-------------------------------+
-| ``SHOW SCHEMAS``                         |                               |
-+------------------------------------------+-------------------------------+
-| ``SHOW TABLES``                          |                               |
-+------------------------------------------+-------------------------------+
-| ``SHOW CREATE TABLE``                    |                               |
-+------------------------------------------+-------------------------------+
-| ``SHOW COLUMNS FROM``                    |                               |
-+------------------------------------------+-------------------------------+
-| ``DESCRIBE``                             | Same as ``SHOW COLUMNS FROM`` |
-+------------------------------------------+-------------------------------+
-| ``CALL kudu.system.add_range_partition`` | Adds range partition to a     |
-|                                          | table. See `Managing range    |
-|                                          | partitions`_                  |
-+------------------------------------------+-------------------------------+
-| ``CALL kudu.system.drop_range_partition``| Drops a range partition       |
-|                                          | from a table. See `Managing   |
-|                                          | range partitions`_            |
-+------------------------------------------+-------------------------------+
-
-``ALTER SCHEMA ... RENAME TO ...`` is not supported.
-
-
-Create table
-------------
+SQL support
+-----------
+
+The connector provides read and write access to data and metadata in
+Kudu. In addition to the :ref:`globally available
+<sql-globally-available>` and :ref:`read operation <sql-read-operations>`
+statements, the connector supports the following features:
+
+* :doc:`/sql/insert`, see also :ref:`kudu-insert`
+* :doc:`/sql/delete`
+* :doc:`/sql/create-table`, see also :ref:`kudu-create-table`
+* :doc:`/sql/create-table-as`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/alter-table`, see also :ref:`kudu-alter-table`
+* :doc:`/sql/create-schema`, see also :ref:`kudu-create-schema`
+* :doc:`/sql/drop-schema`, see also :ref:`kudu-drop-schema`
+
+.. _kudu-insert:
+
+Inserting into tables
+^^^^^^^^^^^^^^^^^^^^^
+
+``INSERT INTO ... values`` and ``INSERT INTO ... select`` behave like
+``UPSERT``.
+
+.. include:: sql-delete-limitation.fragment
```

Comment:
Quick nit, can you move this include above the kudu-insert section and anchor? Having this here looks, at a glance, like the fragment is part of the "inserting into tables" section.

Reply:
@mosabua and I moved it down to match the order of the list.

Reply:
Never mind then!